pkgs/test_core/lib/src/util/package_config.dart

```diff
@@ -29,6 +29,11 @@ Future<Uri> absoluteUri(String path) async {
   final uri = p.toUri(path);
   final absoluteUri =
       uri.isAbsolute ? uri : _originalWorkingDirectory.resolveUri(uri);
-  final packageConfig = await currentPackageConfig;
-  return packageConfig.toPackageUri(absoluteUri) ?? absoluteUri;
+  try {
+    final packageConfig = await currentPackageConfig;
+    return packageConfig.toPackageUri(absoluteUri) ?? absoluteUri;
+  } on StateError catch (_) {
```

Comment:
```suggestion
  } on StateError {
```

Doesn't this work? 🤷 

Not a big deal AT ALL

Reply:
I think that is fine, but I don't want to wait for CI again as I want to get this merged internally to unblock some folks. I will follow up with a fix :)

Reply:
https://github.com/dart-lang/test/pull/2159